### DISCUSSION TITLE
Input field style does'nt work as per docs :: tailwindcss 3.2.2 :: nextjs and typescript

### DISCUSSION THREAD

**Discussion by shamseerahammedm** [tailwindlabs/tailwindcss#9995] [Help] [answered]
using `"tailwindcss": "^3.2.2"` with next js and typescript

default styles for input component not getting applied

https://tailwindui.com/components/application-ui/forms/input-groups

input in the offiicial doc, looks completely different, focus styles also not coming, but all the utlity classes are working any one have any idea ?

globals.css
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```
tailwind.config.js
```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    './src/**/*.{js,ts,jsx,tsx,html}',
    // './pages/**/*.{html,js,tsx}'
    "./pages/**/*.{js,ts,jsx,tsx,html}",
    "./app/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
from docs page : 

![Input Groups - Official Tailwind CSS UI Components 12-3-2022 6-55-41 PM](https://user-images.githubusercontent.com/20143226/205443061-a5e1dd2b-85f3-43b2-971c-8c749708f93b.png)

in my page :

![Dummy 12-3-2022 6-56-14 PM](https://user-images.githubusercontent.com/20143226/205443083-89f877df-6511-4b5d-9a90-47bb0aeb2afe.png)

**Comment by wongjn** [comment, marked as answer]
As the example code states:

> ```html
> 
> ```

So be sure you have the [`@tailwindcss/forms` plugin installed](https://github.com/tailwindlabs/tailwindcss-forms).

**Comment by shamseerahammedm** [comment]
@wongjn thank you that solved the issue, but didnt find it mentioned anywhere in official doc page  ?

https://tailwindcss.com/docs/guides/nextjs

**Reply by wongjn** [reply]
That documentation page is for integrating Tailwind CSS with Next.js, but this is not directly relevant to your situation, since you are talking about Tailwind UI.
